### PR TITLE
Update blog instructions to include Blog Post Starter Pack

### PR DESCRIPTION
**Pull request by zlonko**
This relates to [#5513](https://github.com/sourcegraph/about/issues/5513). 

### Changelog
- Updates guidance for creating a blog post on `/departments/marketing/blog`
- Includes information about the "Starter Pack" blog post template file created with https://github.com/sourcegraph/about/pull/5553

### Test
- Please review the updated instructions on this page

### CODE REVIEW

File: content/departments/marketing/blog.md
Comment (st0nebreaker) [inline content/departments/marketing/blog.md]: I know this file isn't posted yet in the About repo, but it'd be helpful to have a link to the file to be really explicit here

File: content/departments/marketing/blog.md
Comment (st0nebreaker) [inline content/departments/marketing/blog.md]:
```suggestion
The Content Platform Team maintains a template Markdown file that exposes all of the content components available for use in the blog. You can directly copy and paste components from this file into your post. It's important to use the components provided for tables, images, alerts, youtube videos, videos, and hubspot forms to ensure our blog posts are uniform in styling.
```

Some added context

File: content/departments/marketing/blog.md
Comment (st0nebreaker) [inline content/departments/marketing/blog.md]: This is great! Can you add a section for the other missing components here? (`Alert`, `BlockquoteWithBorder`, `TableWrapper`)

**Review by st0nebreaker — CHANGES_REQUESTED** [review]
Thanks Tim! Just a few small comments

File: content/departments/marketing/blog.md
Comment (zlonko) [reply content/departments/marketing/blog.md]: Agreed! I can add it once we merge the About PR.

File: content/departments/marketing/blog.md
Comment (zlonko) [reply content/departments/marketing/blog.md]: Sure thing! I have included descriptions for these three here: 9d453a5

File: content/departments/marketing/blog.md
Comment (st0nebreaker) [inline content/departments/marketing/blog.md]: Do you think this should be a youtube video?

**Review by st0nebreaker — APPROVED** [review]
Just needs @bretthayes approval

File: content/departments/marketing/blog.md
Comment (bretthayes) [inline content/departments/marketing/blog.md]:
```suggestion
The Content Platform Team maintains a template Markdown file that exposes all of the content components available for use in the blog. You can directly copy and paste components from this file into your post. It's important to use the components provided for tables, images, alerts, youtube videos, external or self hosted videos, and HubSpot forms to ensure our blog posts are uniform in styling.
```

File: content/departments/marketing/blog.md
Comment (bretthayes) [inline content/departments/marketing/blog.md]: Instead of referencing all the components code here in the handbook, I think we should just reference the starter pack alone and maintain that as the source of truth. Then we can avoid maintaining two places at once and avoid outdated API mismatches. If we ever change the API on the about site, the blog starter pack will enforce us to update it since it will error/warn on build time. I would then make a note here in this guide to "please reference the blog starter pack." What do you think?

**Review by bretthayes — CHANGES_REQUESTED** [review]

File: content/departments/marketing/blog.md
Comment (st0nebreaker) [reply content/departments/marketing/blog.md]: I see what you mean. So not show the syntax for HubSpot forms either? I can list all the components and point teammates to go to the Blog Post Starter Pack to see the syntax use.

Comment (st0nebreaker) [reply content/departments/marketing/blog.md]: Let me know thoughts on this phrasing! [92083e9](https://github.com/sourcegraph/handbook/pull/4186/commits/92083e927792184ac38b9b139a68ecb2aa741cd4)

Comment (bretthayes) [reply content/departments/marketing/blog.md]: Yeah I wouldn't show the code examples on the Handbook. I would also even move the frontmatter section to the blog starter pack and point people there too.

File: content/departments/marketing/blog.md
Comment (bretthayes) [inline content/departments/marketing/blog.md]: I would update this and move the frontmatter to the blog starter pack and use that as the source of truth for all formatting

File: content/departments/marketing/blog.md
Comment (bretthayes) [inline content/departments/marketing/blog.md]:
```suggestion
To ensure uniformity between commonly used content items, please use the following components where appropriate. These are setup to accommodate screen responsiveness, accessibility concerns, and meet our brand guidelines.
```

**Review by bretthayes — CHANGES_REQUESTED** [review]

File: content/departments/marketing/blog.md
Comment (bretthayes) [inline content/departments/marketing/blog.md]:
```suggestion
To use the template, copy the above file and paste it in the appropriate year folder (i.e. `/content/blogposts/2022/`). Rename the file to the slug the post will be, or something similar. Edit the markdown content accordingly. All available standardized components are ready to be changed right in this file!
```

File: content/departments/marketing/blog.md
Comment (bretthayes) [inline content/departments/marketing/blog.md]:
```suggestion
- Small images can be placed in the `/public/blog` directory and have the url of `/blog/example-image.jpg` in your markdown. We also strongly encourage you to create a sub directory with the same slug as your post if you are including multiple files for your blog post. This keeps our assets organized per blog post. For example: `/blog/my-slug/image-name.png`, `/blog/my-slug/other-image-name.png`.
```

**Review by bretthayes — CHANGES_REQUESTED** [review]
Just a few suggestions and we should be good to publish!

File: content/departments/marketing/blog.md
Comment (bretthayes) [inline content/departments/marketing/blog.md]: The GitHub Action is complaining about this line. I'm thinking this may be resolved by just referencing the anchor link:

```suggestion
If you do this, please add the **frontmatter** to the beginning of the file, frontmatter instructions can be found in the [Blog Post Starter Pack](#using-the-blog-post-starter-pack).
```

**Review by bretthayes — CHANGES_REQUESTED** [review]
I just noticed this is erroring:

File: content/departments/marketing/blog.md
Comment (st0nebreaker) [reply content/departments/marketing/blog.md]: Ah I thought it was failing b/c the link isn't valid in prod yet. Ok this step looks good now, thanks ! 😄

**Review by bretthayes — APPROVED** [review]
🚀

**Pull request merged**
